app/views/projects/_form.html.haml

```diff
@@ -3,8 +3,8 @@
   .form-inputs
     = f.input :name, required: true, autofocus: true
     = f.association :client, required: false, collection: Client.by_name, label: false, placeholder: t("project.form.client"), include_blank: t("project.form.no_client")
-    = f.input :description, as: :text, input_html: { rows: 5 }
     = f.label :billable
-    = f.input :billable, as: :boolean, label: false
+    = f.input :billable, as: :boolean, label: false, disabled: true
```

Comment:
It's a little bit better to disable this with javascript (instead of enabling it), that way users without JS can still use this